outreach/metadata.go

```diff
@@ -0,0 +1,78 @@
+package outreach
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+type ObjectOKResponse struct {
+	Data []map[string]any `json:"data"`
+}
+
+func (c *Connector) ListObjectMetadata(ctx context.Context,
+	objectNames []string,
+) (*common.ListObjectMetadataResult, error) {
+	// Ensure that objectNames is not empty
+	if len(objectNames) == 0 {
+		return nil, common.ErrMissingObjects
+	}
+
+	objMetadata := common.ListObjectMetadataResult{
+		Result: make(map[string]common.ObjectMetadata),
+		Errors: make(map[string]error),
+	}
+
+	for _, obj := range objectNames {
+		// Constructing the  request url.
+		objURL, err := url.JoinPath(c.BaseURL, obj)
+		if err != nil {
+			return nil, err
+		}
+
+		res, err := c.get(ctx, objURL)
+		if err != nil {
+			objMetadata.Errors[obj] = err
+			continue
+		}
+
+		// Check nil response body, to avoid panic.
+		if res == nil || res.Body == nil {
+			objMetadata.Errors[obj] = ErrEmptyResponse
+			continue
```

Comment:
```suggestion

			continue
```

Reply:
@Jkarage the suggestion wasn't to remove `objMetadata.Errors[obj] = ErrEmptyResponse`, it is to simply add a blank line between it and `continue` can you please add the line back?

Reply:
Noted!!